docs/getting_started.md

```diff
@@ -430,6 +463,259 @@ In `provision.yml`, created earlier, we created the deployment config, service,
+      interval_seconds: 1
+      max_surge: 25%
+      max_unavailable: 25%
+      timeout_seconds: 600
+      update_period_seconds: 1
+    spec_template_metadata_labels:
+      app: postgresql-apb
+      service: postgresql
+    containers:
+    - env:
+      - name: POSTGRESQL_PASSWORD
+        value: '{{ postgresql_password }}'
+      - name: POSTGRESQL_USER
+        value: '{{ postgresql_user }}'
+      - name: POSTGRESQL_DATABASE
+        value: '{{ postgresql_database }}'
+      image: docker.io/centos/postgresql-94-centos7
+      image_pull_policy: IfNotPresent
+      name: postgresql
+      ports:
+      - container_port: 5432
+        protocol: TCP
+      resources: {}
+      security_context: {}
+      termination_message_path: /dev/termination-log
+      volume_mounts:
+      - mount_path: /var/lib/pgsql/data
+        name: postgresql
+      working_dir: /
+    dns_policy: ClusterFirst
+    restart_policy: Always
+    termination_grace_period_seconds: 30
+    volumes:
+    - name: postgresql
+      persistent_volume_claim:
+        claim_name: postgresql
+      test: false
+      triggers:
+      - type: ConfigChange
+      
+- name: create service
+  k8s_v1_service:
+    name: postgresql
+    namespace: '{{ namespace }}'
+    state: present
+    labels:
+      app: postgresql-apb
+      service: postgresql
+    selector:
+      app: postgresql-apb
+      service: postgresql
+    ports:
+    - name: port-5432
+      port: 5432
+      protocol: TCP
+      target_port: 5432
+  register: postgres_service      
+```
+Edit the `roles/deprovision-my-pg-apb/tasks/main.yml` and delete the created resources.
+
+[TODO]: # (Find out how to delete the persistent volume.  Find out about leftover replication controllers)
+```yaml
+# roles/deprovision-my-pg-apb/tasks/main.yml
+- k8s_v1_service:
+    name: postgresql
+    namespace: '{{ namespace }}'
+    state: absent
+
+- openshift_v1_deployment_config:
+    name: postgresql
+    namespace: '{{ namespace }}'
+    state: absent
+```
+
+At this point, the APB can create a fully functional Postgres database to our cluster.  However, no other application can bind to it and use it.
+
+##### Bind - Encode credentials
+[TODO]: # (Revisit these instructions in the case we hide the implemetation of the bind better)
+To make the Postgres credentials available to other applications, we need to encode them and then store them so that they'll be available.  Edit the `roles/provision-my-pg-apb/tasks/main.yml` and add the following to the very end:
+```yaml
+- name: encode bind credentials
+  shell: 'echo "{\"POSTGRES_HOST\": \"{{ postgres_service.service.spec.cluster_ip }}\", \"POSTGRES_PORT\": \"5432\", \"POSTGRES_USER\": \"{{ postgresql_user }}\", \"POSTGRES_PASSWORD\": \"{{ postgresql_password }}\", \"POSTGRES_DB\": \"{{ postgresql_database }}\"}" | base64 -w 0'
+  register: encoded_bind_credentials
+
+- debug:
+    msg: "<BIND_CREDENTIALS>{{ encoded_bind_credentials.stdout }}</BIND_CREDENTIALS>"```
```

Comment:
The broker bind output is changing per https://github.com/fusor/apb-examples/pull/25.  So this code will change to https://github.com/fusor/apb-examples/pull/25/commits/f56c0a6bdad4c87fb3a89247c138b874f3974de8